frontend/amp/pages/MainBlock.tsx

```diff
@@ -0,0 +1,273 @@
+import React from 'react';
+import { css, cx } from 'react-emotion';
+import { sans, serif } from '@guardian/pasteup/fonts';
+import { palette } from '@guardian/pasteup/palette';
+import { pillarMap, pillarPalette } from '../../lib/pillars';
+import Dateline from '../../web/components/Dateline';
+import { ShareCount } from '../../web/components/ShareCount';
+import ClockIcon from '@guardian/pasteup/icons/clock.svg';
+import TwitterIcon from '@guardian/pasteup/icons/twitter.svg';
+import { SharingIcons } from '../../web/components/ShareIcons';
+
+// tslint:disable:react-no-dangerous-html
```

Comment:
I'm not sure whether we want to put this at the top of files, top of the component, or even line above (possible by using disable-next-line and adding a JSX comment { // .. } I think.

What do you think? @SiAdcock do we have a standard here?

Reply:
I prefer to use `disable-next-line` above to the line that breaks the rule, so if we refactor the line, we can remove the comment at the same time. If we can use a JSX comment to do this, that would be my ideal solution

Reply:
Yeah it's possible, but a bit confusing sometimes. E.g. this appears to work:

    <span // tslint:disable-line:react-no-dangerous-html
        className={style(pillar)}
        dangerouslySetInnerHTML={{
            __html: html,
        }}
    />

even though the disable rule is on a different line in this case.

Reply:
Yeah, I much prefer the second snippet. Perhaps we should standardise on this. I'll make a note to [update our code style docs](https://github.com/guardian/dotcom-rendering/blob/master/docs/code-style.md)

Reply:
👍

Reply:
I'm just dropping my two cents here:
We don't need it, we have to drop raw html in all over the place and this linter rule is going to be ignored so frequently it might as well not exist.

(Also MS recommend an exceptions file https://github.com/Microsoft/tslint-microsoft-contrib/blob/master/src/reactNoDangerousHtmlRule.ts#L107 :( )